scripts/retrieve_databundle_light.py

```diff
@@ -354,6 +356,85 @@ def download_and_unzip_direct(config, rootpath, hot_run=True, disable_progress=F
     return True
 
 
+def download_and_unzip_hydrobasins(
+    config, rootpath, hot_run=True, disable_progress=False
+):
+    """
+    download_and_unzip_basins(config, rootpath, dest_path, hot_run=True,
+    disable_progress=False)
+
+    Function to download and unzip the data for hydrobasins from HydroBASINS database
+    available via https://www.hydrosheds.org/products/hydrobasins
+
+    We are using data from the HydroSHEDS version 1 database
+    which is © World Wildlife Fund, Inc. (2006-2022) and has been used herein under license.
+    WWF has not evaluated our data pipeline and therefore gives no warranty regarding its
+    accuracy, completeness, currency or suitability for any particular purpose.
+    Portions of the HydroSHEDS v1 database incorporate data which are the intellectual property
+    rights of © USGS (2006-2008), NASA (2000-2005), ESRI (1992-1998), CIAT (2004-2006),
+    UNEP-WCMC (1993), WWF (2004), Commonwealth of Australia (2007), and Her Royal Majesty
+    and the British Crown and are used under license. The HydroSHEDS v1 database and
+    more information are available at https://www.hydrosheds.org.
+
+    Inputs
+    ------
+    config : Dict
+        Configuration data for the category to download
+    rootpath : str
+        Absolute path of the repository
+    hot_run : Bool (default True)
+        When true the data are downloaded
+        When false, the workflow is run without downloading and unzipping
+    disable_progress : Bool (default False)
+        When true the progress bar to download data is disabled
+
+    Outputs
+    -------
+    True when download is successful, False otherwise
+    """
+    resource = config["category"]
+    url_templ = config["urls"]["hydrobasins"]
+    suffix_list = config["urls"]["suffixes"]
+
+    level_code = snakemake.config["renewable"]["hydro"]["resource"]["hydrobasins_level"]
+    if level_code <= 9:
+        level_code = "0" + str(level_code)
+    else:
+        level_code = str(level_code)
```

Comment:
Could be replaced with
```suggestion
    level_code = "{:02d}".format(int(level_code))
```

Reply:
Great solution! Implemented.